.github/workflows/ci.yaml

```
@@ -101,7 +101,7 @@ jobs:
         pytest --doctest-modules openmmforcefields --ignore=openmmforcefields/tests
 
     - name: Upload coverage report to CodeCov
-      uses: codecov/codecov-action@v3.1.1
+      uses: codecov/codecov-action@v3.1.4
```

```suggestion
      uses: codecov/codecov-action@v3
```

This makes dependabot less noisy but still gets the job done